### PR TITLE
Feature/userstory 8

### PR DESCRIPTION
**Pull request by AstridOttmann**
Implemented a search function on both of the pages.

### CODE REVIEW

File: components/SearchForm/index.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+      </StyledForm>
+      {searchResults.length === 0 && searchInput.length > 0 ? (
+        <StyledSection>
+          <StyledMessage>Not found</StyledMessage>
+        </StyledSection>
+      ) : null}
+      {searchInput.length > 0 && (
+        <StyledList>
+          {searchResults
+            .filter((translation) => translation.word.includes(searchInput))
```
Comment (niklasrammelkamp) [inline components/SearchForm/index.js]: Why do you filter here? I would think that `searchResults` already contains all results and doesn't need to filter again. 
And also this filter does the same thing as what your filter in line 24 does, I think.

File: components/SearchForm/index.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+                    setSearchInput(translation.word);
+                  }}
+                >
+                  <StyledMessage>
+                    {translation.word} -{" "}
+                    {!selectedLanguage ? (
+                      <small>({translation.language}) - </small>
+                    ) : (
+                      ""
+                    )}
```
Comment (niklasrammelkamp) [inline components/SearchForm/index.js:67]: I think you always should display the language no matter if I selected a language or not

File: pages/index.js [hunk 3]
```diff
@@ -49,29 +52,31 @@ export default function HomePage() {
...
-          isEditMode={false}
-          onSubmitEvent={handleAddTranslation}
-          onFirstInput={handleFirstInput}
-        />
+        <StyledTitle>Add word</StyledTitle>
         <StyledSection>
-          {isFound && <p>word already exists</p>}
+          {isFound && <StyledMessage>word already exists</StyledMessage>}
           {isFound === false && (
             <>
```
Comment (niklasrammelkamp) [inline pages/index.js:58]:
```suggestion
          {isFound ? <StyledMessage>word already exists</StyledMessage> : (
            <>
```

**Review by niklasrammelkamp — COMMENTED** [review]
Well done. 🦾 really nice. I like. 
Only have three small points.

File: components/SearchForm/index.js (hunk 1 above)
Comment (AstridOttmann) [reply components/SearchForm/index.js]: You are right, I removed the filter in line 49.

**Pull request merged**